index.html

```diff
@@ -2,100 +2,90 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Sean Twomey's Portfolio</title>
+    <!-- Title shown when user hovers over the tab and title bar -->
+    <title>Sean Twomey's Personal Website</title>
     <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300&display=swap"/>
```

Comment:
Perhaps make this formatting change across all files (if the code does not end up getting reused).